internal/signerverifier/dsse/dsse.go

```diff
@@ -56,7 +56,7 @@ func SignEnvelope(ctx context.Context, envelope *dsse.Envelope, signer dsse.Sign
 }
 
 // VerifyEnvelope verifies a DSSE envelope against an expected threshold using
-// a slice of verifiers passed in to it. Threshold indicates the number of
+// a slice of verifiers passed into it. Threshold indicates the number of
```

Comment:
`in to` can be correct if the thing after `to` is a verb, but it isn't in this case.